extensions/src/test/java/org/pentaho/test/platform/web/http/security/RequestParameterAuthenticationFilterTest.java

```diff
@@ -97,4 +115,157 @@ public void userNamePasswordUnencrypted() throws IOException, ServletException {
     verify( authManagerMock ).authenticate( Mockito.eq( authRequest ) );
 
   }
+
+
+  @Test
+  public void testSetCsrfProcessorRespectsIt() {
+
+    CsrfProcessor csrfProcessorMock = mock( CsrfProcessor.class );
+    RequestParameterAuthenticationFilter filter = new RequestParameterAuthenticationFilter();
+
+    // ---
+
+    filter.setCsrfProcessor( csrfProcessorMock );
+
+    // ---
+
+    assertSame( csrfProcessorMock, filter.getCsrfProcessor() );
+  }
+
+  private void testRequestParameterAuthenticationFailsThenThrowsCsrfValidationAuthException( @NonNull Throwable validateError )
```

Comment:
This name should have been changed according to the same rules used for others.
E.g. `testWhenCsrfValidationFailsWithGivenExceptionThenRethrows`.